interfaces/python/CMakeLists.txt

```diff
@@ -7,284 +7,272 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-if(HELICS_SWIG_GENERATE_INTERFACE_FILES_ONLY)
-    include(${CMAKE_CURRENT_SOURCE_DIR}/pythonSwigGenerateOnly.cmake)
-else()
+cmake_minimum_required(VERSION 3.4)
+project(helicsPYTHON)
```

Comment:
should this have a version associated with it?

Reply:
Since the only way it is distributed is in the HELICS repository and source tarballs, I think that gives pretty strong ties to a particular HELICS version. If we add a separate version for the Python interface we'd need to figure out what version numbers to use (every time swig updates the .c wrapper file?).

Reply:
I was assuming it would be the same the HELICS_VERSION, I just don't want CMAKE to spit out a warning if nothing is there.  Not sure if it does that on Newer cmake or not.  If not I am fine leaving it off.

Reply:
The docs say the VERSION argument is optional and needs CMP0048 set to new — it mentions 3.16.2+ will output a warning if VERSION is used and the policy is not set (seems to be for compatibility with pre-3.0 projects that manage version variables themselves).